webpack/ForemanTasks/Components/TasksTable/Components/ConfirmationModals.js

```diff
@@ -1,55 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  CANCEL_CONFIRM_MODAL_ID,
-  RESUME_CONFIRM_MODAL_ID,
-  CANCEL_SELECTED_CONFIRM_MODAL_ID,
-  RESUME_SELECTED_CONFIRM_MODAL_ID,
+  CONFIRM_MODAL,
+  CANCEL_MODAL,
+  RESUME_MODAL,
 } from '../TasksTableConstants';
-import { CancelConfirm } from './CancelConfirm';
-import { ResumeConfirm } from './ResumeConfirm';
+import { ConfirmModal } from './ConfirmModal';
 
 export const ConfirmationModals = ({
-  modalProps,
+  setModalClosed,
   tasksActions,
   selectedRowsLen,
+  modalID,
 }) => (
   <React.Fragment>
-    <CancelConfirm
-      closeModal={modalProps.cancelModal.setModalClosed}
-      action={tasksActions.cancelTask}
-      selectedRowsLen={1}
-      id={CANCEL_CONFIRM_MODAL_ID}
-    />
-    <ResumeConfirm
-      closeModal={modalProps.resumeModal.setModalClosed}
-      action={tasksActions.resumeTask}
-      selectedRowsLen={1}
-      id={RESUME_CONFIRM_MODAL_ID}
-    />
-    <CancelConfirm
-      closeModal={modalProps.cancelSelectedModal.setModalClosed}
-      action={tasksActions.cancelSelectedTasks}
-      selectedRowsLen={selectedRowsLen}
-      id={CANCEL_SELECTED_CONFIRM_MODAL_ID}
-    />
-    <ResumeConfirm
-      closeModal={modalProps.resumeSelectedModal.setModalClosed}
-      action={tasksActions.resumeSelectedTasks}
-      selectedRowsLen={selectedRowsLen}
-      id={RESUME_SELECTED_CONFIRM_MODAL_ID}
+    <ConfirmModal
```

Comment:
since we are just rendering `ConfirmModal` in `ConfirmationModals`,
can we avoid this file and render `ConfirmModal` in the parent component?

Reply:
I'll think it's better to make a new file to split some of the logics, also this file will have more logic in the next PR so it's going to be needed for that.

Reply:
I think the best approach would be to have all of the logic in the parent component,
and than to have very simple component which mostly handle the UI elements rendering,

we also won't need extra wrappers, and components with similar names such as `ConfirmationModals` and `ConfirmModal`, and the props that we pass will be more simple to understand.

Reply:
Yeah that makes sense, changed